### ISSUE TITLE
App Crash in android 8

### ISSUE THREAD

**Issue by tundisi** [Buglife/buglife-android#41]
Hello, Good Day.

I'm having problems on Android 8 when I try to use bug life, but all in samgsum devices, I hope you can help me solve it.

![screen shot 2018-06-15 at 10 21 34 am](https://user-images.githubusercontent.com/16090751/41472963-e92405bc-7085-11e8-9dc8-e175244a00a6.png)

**Comment by schukin** [comment]
Hi - what is your `compileSdkVersion` and `targetSdkVersion`?

**Comment by joejune** [comment]
I'm seeing similar crashes reported as well.

```
    minSdkVersion = 21
    targetSdkVersion = 27
    compileSdkVersion = targetSdkVersion
```
![crashes](https://user-images.githubusercontent.com/23319269/41928448-1202e456-792a-11e8-8be3-25eb0371e45c.png)

**Comment by tundisi** [comment]
hi my compileSdkVersion is 27, buildToolsVersion '26.0.2' , targetSdkVersion 26

**Comment by joejune** [comment]
Looks like this may be fixed in the 1.3.1 release, specifically 9d40de4.  I am trying this now.

**Comment by tundisi** [comment]
Hi @joejune, did it work for you?

**Comment by joejune** [comment]
@tundisi: Yes, absolutely.  I went from many thousands of crashes per user to zero.  If you look at the crash stack trace and the code change in 9d40de4, it's quite clear that it fixes the cause of the crash.

If I had reported this issue, I would mark it as resolved.

**Comment by tundisi** [comment]
thanks @joejune , I'll do the test and comment if I get some error

**Comment by tundisi** [comment]
Hello everyone

I write this as a help to be able to use buglife without problems.

Initially added to the `com.buglife.sdk library: buglife-android: 1.3.1`, getting an error that does not find volley. Looking for errors I found that I asked to degrade to version `1.2.4` and in that version I got the error found in this thread (Solved in version 1.3.1).

In order to run version 1.3.1 you must include the implementation of the application `implementation 'com.android.volley:volley:1.1.0'` and add these permissions in the maninfest

```
<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
<uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
<uses-permission android:name="android.permission.INTERNET" />
```

I hope it will help many.

A hug,
regards

PD: thnx @joejune @schukin
